src/app.rs

```diff
@@ -254,9 +271,24 @@ impl App {
     where
         B: Backend,
     {
-        let chunks = Layout::default()
-            .constraints([Constraint::Percentage(100)].as_ref())
-            .split(f.size());
+        let chunks: Vec<Rect>;
+        if self.rosout_widget_enabled {
+            assert!(self.rosout_screen_percentage <= 100);
+            let robot_view_percentage = 100 - self.rosout_screen_percentage;
+            chunks = Layout::default()
+                .constraints(
+                    [
+                        Constraint::Percentage(robot_view_percentage),
+                        Constraint::Percentage(self.rosout_screen_percentage),
+                    ]
+                    .as_ref(),
+                )
+                .split(f.size());
```

Comment:
You need to change the size factors here to reflect the 80%
        let scale_factor = self.terminal_size.0 as f64 / self.terminal_size.1 as f64 * 0.5;
Here you need to add another multiplier

Reply:
Done. The y-scaling had to changed: https://github.com/carzum/termviz/pull/43/commits/27e1c3cf0914bc86f46aed6a112ae983a0cf7ac4